orbit/README.md

```diff
@@ -167,6 +167,14 @@ This process may take several minutes to complete as the Notarization process co
 
 After successful notarization, the generated "ticket" is automatically stapled to the package.
 
+#### Orbit Osquery Result and Status Logs
+
+If the `logger_path` configuration is set to `filesystem`, Orbit will store osquery's "result" and
+"status" logs to the following directories:
```

Comment:
nitpick, are quotes around "result" and "status" necessary?

Reply:
Maybe not, just wanted to mark them as keywords defined like that by osquery.